src/intrprtr.c

```diff
@@ -47,58 +47,21 @@
 #include "hpc/guards.h"
 #endif
 
-/****************************************************************************
-**
-*V  IntrReturning   . . . . . . . . . . .  interpreter is currently returning
-**
-**  If 'IntrReturning' is  non-zero, the interpreter is currently  returning.
-**  The interpreter switches  to this mode when  it finds a return-statement.
-**  If it interprets a return-value-statement, it sets 'IntrReturning' to 1.
-**  If it interprets a return-void-statement,  it sets 'IntrReturning' to 2.
-**  If it interprets a quit-statement, it sets 'IntrReturning' to 8.
-*/
-/* TL: UInt IntrReturning; */
-
-
-/****************************************************************************
-**
-*V  IntrIgnoring  . . . . . . . . . interpreter is currently ignoring actions
-**
-**  If 'IntrIgnoring'  is  non-zero,  the interpreter  is  currently ignoring
-**  actions.  The interpreter switches to this mode for  the right operand of
-**  'or' and 'and'  constructs where the  left operand already determines the
-**  outcome.
-**
-**  This mode is also used in Info and Assert, when arguments are not printed.
-*/
-/* TL: UInt IntrIgnoring; */
-
-
-/****************************************************************************
-**
-*V  IntrCoding  . . . . . . . . . . . interpreter is currently coding actions
-**
-**  If 'IntrCoding' is non-zero, the interpreter is currently coding actions.
-**  The interpreter  switches  to this  mode for  constructs  that it  cannot
-**  directly interpret, such as loops or function bodies.
-*/
-/* TL: UInt IntrCoding; */
-
```

Comment:
The above comments are all already in the header file, so they can be removed here.